app/src/main/java/com/sun/hero_02/data/model/CatagoryHero.kt

```diff
@@ -0,0 +1,72 @@
+package com.sun.hero_02.data.model
+
+import com.sun.hero_02.R
+import com.sun.hero_02.utils.HeroDifficulty
+import com.sun.hero_02.utils.HeroGroup
+
+data class CategoryHero (
+    val image: Int,
+    val nameGroup: String
+)
+
+object HeroGroups {
+
+    enum class imageGroups(val icon: Int) {
+        CLASS(R.drawable.ic_all_class),
+        ASSASSIN(R.drawable.ic_assassin),
+        FIGHTER(R.drawable.ic_fighter),
+        MAGE(R.drawable.ic_mage),
+        TANK(R.drawable.ic_tank),
+        SUPPORT(R.drawable.ic_support),
+        MARKSMAN(R.drawable.ic_marksman)
+    }
+
+    enum class heroGroups(val group: String){
+        CLASS(HeroGroup.ANY_CLASS.tag),
+        ASSASSIN(HeroGroup.ASSASSIN.tag),
+        FIGHTER(HeroGroup.FIGHTER.tag),
+        MAGE(HeroGroup.MAGE.tag),
+        TANK(HeroGroup.TANK.tag),
+        SUPPORT(HeroGroup.SUPPORT.tag),
+        MARKSMAN(HeroGroup.MARKSMAN.tag)
+    }
+
+    fun getHeroClasses() : MutableList<CategoryHero> {
+        val listClasses = mutableListOf<CategoryHero>()
+        for (i in imageGroups.values().indices) {
+            val imageID = imageGroups.values()[i]
+            val heroClassName = heroGroups.values()[i]
+            listClasses.add(CategoryHero(imageID.icon, heroClassName.group))
+        }
+        return listClasses
```

Comment:
sao không gộp lại mà e lại tách ra 2 class xong rồi lại chạy for gộp lại chúng nhỉ?